src/jit/morph.cpp

```diff
@@ -2799,8 +2799,9 @@ GenTreeCall* Compiler::fgMorphArgs(GenTreeCall* callNode)
         // this can't be a struct.
         assert(argx->gtType != TYP_STRUCT);
 
+        // FIXME: Why do we need floating type for 'this' argument
```

Comment:
Nit: it would be nice to have an issue tracking this FIXME if it's still relevant. Would you mind opening one and tagging this comment with the issue number?